lib/report_formatter/timeline.rb

```diff
@@ -175,25 +175,6 @@ def tl_event(tl_xml, row, col)
       # manipulating column order to display timestamp at the end of the bubble.
       field = mri.timeline[:field].split("-")
       if ems && ems_cloud
-        # Remove Infra specific fields
```

Comment:
@moolitayer why is this piece of code being removed, wont this cause Infra specific fields to show up on cloud specific Timelines.